### ISSUE TITLE
Conv layers before BatchNormalization layers should not have biases

### ISSUE THREAD

**Issue by bersbersbers** [ellisdg/3DUnetCNN#218]
With `batch_normalization=True`, the `Conv3D` layers followed by a `BatchNormalization` layers could/should use `use_bias=False`.
